playwright/e2e/crypto/utils.ts

```diff
@@ -155,23 +155,41 @@ export async function checkDeviceIsConnectedKeyBackup(
         );
     }
 
-    await page.getByRole("button", { name: "User menu" }).click();
-    await page.locator(".mx_UserMenu_contextMenu").getByRole("menuitem", { name: "Security & Privacy" }).click();
-    await expect(page.locator(".mx_Dialog").getByRole("button", { name: "Restore from Backup" })).toBeVisible();
+    const backupData = await app.client.evaluate(async (client: MatrixClient) => {
+        const crypto = client.getCrypto();
+        if (!crypto) return;
 
-    // expand the advanced section to see the active version in the reports
-    await page.locator(".mx_SecureBackupPanel_advanced").locator("..").click();
+        const backupInfo = await crypto.getKeyBackupInfo();
+        const backupKeyStored = Boolean(await client.isKeyBackupKeyStored());
```

Comment:
```suggestion
        const backupKeyIn4S = Boolean(await client.isKeyBackupKeyStored());
```

Reply:
In this case, it's somewhat implied that it's the private key (since it's in 4S), but maybe call this `backupPrivateKeyIn4S` for consistency.